File: passwordvault/src/main/java/co/blustor/pwv/utils/AlertUtils.java
package co.blustor.pwv.utils;

import android.app.AlertDialog;
import android.content.Context;

public class AlertUtils {
    public static void showError(Context context, String message) {
        showMessage(context, message);
    }

    private static void showMessage(Context context, String message) {
        AlertDialog.Builder builder = new AlertDialog.Builder(context)
                .setMessage(message)
                .setPositiveButton("Okay", (dialog, which) -> dialog.cancel());

        builder.create().show();
    }
}

File: gatekeeper/src/main/java/co/blustor/identity/utils/AlertUtils.kt
package co.blustor.identity.utils

import android.app.AlertDialog
import android.content.Context

object AlertUtils {

    private fun showMessage(context: Context, title: String, message: String) {
        val builder = AlertDialog.Builder(context).setMessage(message)
            .setTitle(title)
            .setPositiveButton("Okay") { dialog, _ ->
                dialog.cancel()
            }

        builder.create().show()
    }

    fun showError(context: Context, message: String) {
        showMessage(context, "Error", message)
    }
}
